### ISSUE TITLE
Store audio features instead of popularity

### ISSUE THREAD

**Issue by tandy-1000** [krestaino/sptnr#5]
Hi, really cool tool, thanks for sharing!

I DJ as a hobby and I've heard of DJs using the star rating system to determine track energy.

Spotify has an [audio features API](https://developer.spotify.com/documentation/web-api/reference/get-audio-features) which has information such as energy, and danceability.

It would be cool if your tool would allow you to set which datapoint is stored as a star rating.

This would be even more useful if the rating is stored in the file metadata and not just the Navidrome DB as you could utitilise it in DJ software.

Would you be open to this?
Thanks!

**Comment by Kixel** [comment]
I'd love this feature as well, but as far as I can tell, navidrome, or rather subsonic-API, does not have any metadata fields for this. The only (API) thing i could find was for staring and rating:
![grafik](https://github.com/krestaino/sptnr/assets/1922049/b0e421c7-5b82-441b-817b-6dea3b8b53a4)
(see: [https://www.navidrome.org/docs/developers/subsonic-api/](https://www.navidrome.org/docs/developers/subsonic-api/) )

**Comment by tandy-1000** [comment]
Yes, that's true, but I'd be happy with a configuration setting which lets you choose which audio feature is stored as a star rating, e.g. I would like energy.
